components/genesys-email-flow/modules/email_routes/main.tf

```diff
@@ -0,0 +1,40 @@
+terraform {
+  required_providers {
+    genesyscloud = {
+      source = "mypurecloud/genesyscloud"
+
+    }
+  }
+}
+
+#
+# Description:  
+#
+# Last step in the process.  We are going to create the email domain and route.  Since this has to happen after the archy flow,
+# we explicitly create a dependency on the archy flow.
+
+
+resource "genesyscloud_routing_email_domain" "devengage_email_domain" {
```

Comment:
Genesys Cloud only allows two email domains by default. This is the error encountered by @AgnesCorpuz . It might be a good idea to also implement email domains as data sources in terraform so we could use an exiting one.

Reply:
Oh, but for now in the blueprint, we might have to include a text saying that they should have an available slot for email domain.

Reply:
Reply good call on adding the data source.  I will add a ticker for a data source

Reply:
Added some commentary to the script